tests/postgres_tests/test_array.py

```diff
@@ -243,6 +243,9 @@ def test_usage_in_subquery(self):
             [self.objs[3]]
         )
 
+    def test_ticket25666(self):
```

Comment:
Could you give the test method a more descriptive name and add a reference to the ticket in a comment instead?